### ISSUE TITLE
Extension does not find appropriate code actions

### ISSUE THREAD

**Issue by ivanarellano** [mjbvz/vscode-folder-source-actions#1]
Extension tested on:
Mac 10.13.6
VSCode 1.26 and Insiders 1.27 (b00ab5288f314f6ef8d82923d89a0555ad15eb06)

Thank you for starting this extension. I'm running this on a project with several unused imports in JS/JSX files. As I understand it, this extension runs the command `editor.action.organizeImports` on a directory. My problem is that nothing appears to happen when I click "Organize Imports in Folder".

The function `getAllCodeActionsForFile` returns an empty array which doesn't seem right at all. Also, I'm not seeing that the command `vscode.executeCodeActionProvider` exists within a list from `vscode.commands.getCommands(false)` that's executed in the extension. This is my filtered output of getCommands names prefixed with vscode:

```
vscode.previewHtml
vscode.openFolder
vscode.diff
vscode.open
vscode.removeFromRecentlyOpened
vscode.setEditorLayout
vscode.previewHtml
vscode.openFolder
vscode.diff
vscode.open
vscode.removeFromRecentlyOpened
vscode.setEditorLayout
```

I'm entirely new to Code extension development and hope that this helps. I'm unsure where to look to fix this.

**Comment by mjbvz** [comment]
Are you sure there are files with organizable imports in your folder? Can you run the organize imports command in them individually?

**Comment by ivanarellano** [comment]
Hi, sorry for the late reply! Yes, I can run the command on them. I went through around 300 files that day and ran the default "Shift+Alt+O" shortcut to manually tidy up. I sometimes also ran the lightbulb shortcut with the desired "Remove imports from [package]".

Today I tested this again with Code 1.26.1. I saved an intentionally made unused import and the extension does not seem to run any command.
